content/integrations/integrating-npm-with-external-services/docker-and-private-modules.mdx

```diff
@@ -4,87 +4,71 @@ redirect_from:
   - /private-modules/docker-and-private-modules
 ---
 
-To install private npm packages in a Docker container, you will need to use Docker's build-time variables.
+To install private npm packages in a Docker container, you will need to use [Docker build secrets](https://medium.com/learnwithrahul/securely-using-npm-credentials-with-docker-f38b43bdb7c1).
```

Comment:
Can we link to docker's official documentation here rather than a 3rd party blog?

I found https://docs.docker.com/engine/swarm/secrets/, but unsure if it is correct

Reply:
Committed the suggestion - thanks!

Reply:
I actually think this would be a better reference link 
```suggestion
To install private npm packages in a Docker container, you will need to use [Docker build secrets](https://docs.docker.com/develop/develop-images/build_enhancements/#new-docker-build-secret-information).
```